src/doc/unstable-book/src/compiler-flags/extern options.md

```diff
@@ -0,0 +1,19 @@
+The behavior of the `--extern` flag can be modified with `noprelude`, `priv` or `nounused` options.
+
+This is unstable feature, so you have to provide `-Zunstable-options` to get this feature working.
```

Comment:
Is this true for all the modifiers, or just noprelude?

```suggestion
This is unstable feature, so you have to provide `-Zunstable-options` to enable it.
```

Reply:
Seems to be true of all modifiers:

```
Z:\test> rustc main.rs --extern priv:core=libcore.rlib
error: the `-Z unstable-options` flag must also be passed to enable `--extern options
```